morpheus/stages/postprocess/add_classifications_stage.py

```diff
@@ -67,7 +67,9 @@ def __init__(self, c: Config, threshold: float = 0.5, labels: typing.List[str] =
                 logger.warning("The label '%s' is not in Config.class_labels and will be ignored", label)
                 continue
 
-            self._idx2label[self._class_labels.index(label)] = self._prefix + label
+            prefixed_label = self._prefix + label
+            self._idx2label[self._class_labels.index(label)] = prefixed_label
+            self.needed_columns[prefixed_label] = '?'
```

Comment:
It's numpy's type string for a boolean field (although 'bool' also works), I went with '?' over 'bool' since our `DataType` class was already using '?' as the type_char.